Cargo.toml

```diff
@@ -17,10 +17,10 @@ complex = ["num-complex", "num-traits", "pyo3/num-complex"]
 extension-module = ["pyo3/extension-module"]
 
 [dependencies]
-num-complex = { version = "0.4.0", optional = true }
-num-traits = { version = "0.2.15", optional = true }
```

Comment:
I don't think the `^` makes any difference in cargo

Reply:
Yeah, it doesn't. I went ahead and removed them.